scripts/en2ja.json

```diff
@@ -0,0 +1,104 @@
+{
+  "title": "English-to-Japanese string replacement",
+  "description": "Replace EN string 'from' with associated JA string 'to'.",
+  "pairs": [
+    {
+      "from": "rel=\"canonical\" href=\"https://help.okta.com/asa/en-us",
+      "to": "rel=\"canonical\" href=\"https://help.okta.com/asa/ja-jp",
+      "note": "Many (but not all) topics contain a canonical link in the header."
+    },
+    {
+      "from": "rel=\"canonical\" href=\"https://help.okta.com/en-us",
+      "to": "rel=\"canonical\" href=\"https://help.okta.com/ja-jp",
+      "note": "Many (but not all) topics contain a canonical link in the header."
+    },
+    {
+      "from": "rel=\"canonical\" href=\"https://help.okta.com/eu/en-us",
+      "to": "rel=\"canonical\" href=\"https://help.okta.com/eu/ja-jp",
+      "note": "Many (but not all) topics contain a canonical link in the header."
+    },
+    {
+      "from": "rel=\"canonical\" href=\"https://help.okta.com/oag/en-us",
+      "to": "rel=\"canonical\" href=\"https://help.okta.com/oag/ja-jp",
+      "note": "Many (but not all) topics contain a canonical link in the header."
+    },
+    {
+      "from": "rel=\"canonical\" href=\"https://help.okta.com/oie/en-us",
+      "to": "rel=\"canonical\" href=\"https://help.okta.com/oie/ja-jp",
+      "note": "Many (but not all) topics contain a canonical link in the header."
+    },
+    {
+      "from": "rel=\"canonical\" href=\"https://help.okta.com/wf/en-us",
+      "to": "rel=\"canonical\" href=\"https://help.okta.com/wf/ja-jp",
+      "note": "Many (but not all) topics contain a canonical link in the header."
+    },
+    {
+      "from": "\"https://help.okta.com/en-us/Content/index.htm\"",
+      "to": "\"https://help.okta.com/ja-jp/Content/index.htm\"",
+      "note": "This link is in the top-level menu bar in all topics."
+    },
+    {
+      "from": "\"https://help.okta.com/en-us/Content/Topics/ReleaseNotes/okta-relnotes.htm\"",
+      "to": "\"https://help.okta.com/ja-jp/Content/Topics/ReleaseNotes/okta-relnotes.htm\"",
+      "note": "This link is in the top-level menu bar in all topics."
+    },
+    {
+      "from": "\"https://support.okta.com/help/s/\"",
+      "to": "\"https://support.okta.com/help/s/?language=ja\"",
+      "note": "Apply query param to direct users to their language."
+    },
+    {
+      "from": "?language=en_US",
+      "to": "?language=ja",
+      "note": "Query param as used by various Okta Support assets. Appears in multiple links."
+    },
+    {
+      "from": "alt=\"Okta Support\"",
+      "to": "alt=\"Oktaサポート\"",
```

Comment:
Can you remind me why we need this? Ideally the content we get from XTM has this translations, this should not live in this file in my opinion, it should be a task for XTM

Reply:
We're in a mixed state currently (such as with aria-label rules), once we get harmonized then we can remove this rule.

Reply:
I'm ok with a one-time fix, but we should remove them before we run the script again
https://oktainc.atlassian.net/browse/OKTA-548618